modules/@apostrophecms/attachment/index.js

```diff
@@ -694,11 +694,11 @@ module.exports = {
             // apos.attachment.url with the returned object
             for (let i = ancestors.length - 1; i >= 0; i--) {
               const ancestor = ancestors[i];
-              const fields = ancestor.imagesFields && ancestor.imagesFields[o._id];
```

Comment:
A bug was there, thanks to @boutell's help, we changed it to make it work.

- `imagesFields` did not exist anymore, it was changed to `imageFields`
- passing the image document in `widget.html` (via `{% set attachment = apos.image.first(data.widget._image) %}`), would not even work with the right `imageFields` because no ancestors would have this property.

So we could have passed the entire widget instead (that has the property), but we decided that it would be clearer to pass the image and use its fields `_fields` directly to retrieve the cropping and focal point values